src/js/views/globals/prelogin/prelogin.scss

```diff
@@ -0,0 +1,107 @@
+body {
+  margin: 0;
+}
+
+.prelogin * {
```

Comment:
I bet this is covered by the core scss as well.

Reply:
After importing `'scss/provider-core.scss'`, I was able to remove this `.prelogin *` along with several other properties in the file that were duplicates from the core scss.